src/DotNetLightning.Core/Channel/CommitmentsModule.fs

```diff
@@ -479,33 +526,18 @@ module ForceCloseFundsRecovery =
         let delayedPaymentPrivKey =
             perCommitmentPoint.DeriveDelayedPaymentPrivKey
                 localChannelPrivKeys.DelayedPaymentBasepointSecret
+
+        let coinOptions = CoinOptions()
+        coinOptions.Sequence <- Nullable <| Sequence(uint32 commitments.LocalParams.ToSelfDelay.Value)
```

Comment:
use property initialization to avoid mutation:

```
let coinOptions = CoinOptions (Sequence = Nullable <| Sequence(uint32 commitments.LocalParams.ToSelfDelay.Value))
```